source/clear-linux/tutorials/multi-boot/multi-boot-mint.rst

```diff
@@ -0,0 +1,145 @@
+   steps.
+
+   #. Under :guilabel:`Device` column, select :file:`/dev/sda2`.
+
+   #. Click :guilabel:`Change`.
+
+   #. Confirm :guilabel:`Use as` is set to :guilabel:`Swap area`. See Figure 4.
+
+      .. figure:: figures/multi-boot-mint-4.png
+
+         Figure 4: Mint: Set swap partition
+
+#. Follow the remaining prompts to complete the installation of Mint.
+
+#. At this point, the ability to boot |CL| is lost because `Grub`
+   is the default boot loader. Follow these steps to make the |CL|
+   Systemd-Boot the default boot loader and add Mint as a boot option.
+
+   #. Boot into Mint.
+
+   #. Log in.
+
+   #. Get root permissions.
+
+      .. code-block:: console
+
+         $ sudo -s
+
+   #. Locate the Mint :file:`grub.cfg` file in the :file:`/boot/grub/` and
+      look for the :guilabel:`menuentry` section. The highlighted lines
+      identify the kernel, the :file:`initrd` files, the root partition UUID,
+      and the additional parameters used. Use this information to create a
+      new Systemd-Boot entry for Mint. See Figure 5.
+
+      .. figure:: figures/multi-boot-mint-5.png
+
+         Figure 5: Mint: grub.cfg
+
+   #. Copy the kernel and :file:`initrd` to the EFI partition.
+
+      .. code-block:: console
+
+         # cp /boot/vmlinuz-4.4.0-53-generic /boot/efi
+
+         # cp /boot/initrd.img-4.4.0-53-generic /boot/efi
+
+   #. Create a boot entry for Mint. The file must contain at least these
+      settings:
+
+      +---------+------------------------------------+
+      | Setting | Description                        |
+      +=========+====================================+
+      | title   | Text to show in the boot menu      |
+      +---------+------------------------------------+
+      | linux   | Linux kernel image                 |
+      +---------+------------------------------------+
+      | initrd  | initramfs image                    |
+      +---------+------------------------------------+
+      | options | Options to pass to the EFI program |
+      |         | or kernel boot parameters          |
+      +---------+------------------------------------+
+
+      See the `systemd boot loader documentation`_ for additional
+      details.
+
+      The *options* parameters must specify the root partition UUID and
+      any additional parameters that Mint requires.
+
+      .. note:: The root partition UUID used below is unique to this example.
+
+      .. code-block:: console
+
+         # cd /boot/efi/loader/entries
+
+         # vi mint.conf
+
+      Add the following lines to the :file:`mint.conf` file:
+
+      .. code-block:: console
+
+         title Mint 18.1 Serena MATE
+
+         linux /vmlinuz-4.4.0-53-generic
+
+         initrd /initrd.img-4.4.0-53-generic
+
+         options root=UUID=af4901e1-6238-470a-8c14-bc0f0f7715ec ro
+
+#. Re-install Systemd-Boot to make it the default boot loader.
+
+   .. code-block:: console
+
+      # bootctl install --path /boot/efi
```

Comment:
Change "# bootctl install --path /boot/efi" to "$ sudo bootctl install --path /boot/efi"

Reply:
Text edits deferred to next phase of updates.